server/etcdserver/api/etcdhttp/health.go

```diff
@@ -252,6 +253,8 @@ func installReadyzEndpoints(lg *zap.Logger, mux *http.ServeMux, server ServerHea
 	reg.Register("serializable_read", readCheck(server, true))
 	// linearizable_read check would be replaced by read_index check in 3.6
 	reg.Register("linearizable_read", readCheck(server, false))
+	// check if local is learner
+	reg.Register("learner", learnerCheck(server))
```

Comment:
Let's be clearer. `/readyz/learner` isn't clear what's it's for.

```suggestion
	reg.Register("non-learner", learnerCheck(server))
```

Reply:
> Let's be clearer. `/readyz/learner` isn't clear what's it's for.

done